libkbfs/folder_block_ops.go

```diff
@@ -290,18 +279,19 @@ func (fbo *folderBlockOps) getBlockHelperLocked(ctx context.Context,
 		return nil, InvalidBlockRefError{ptr.Ref()}
 	}
 
-	if block, err := fbo.getBlockFromDirtyOrCleanCache(
-		ptr, branch); err == nil {
+	if block, err := fbo.config.DirtyBlockCache().Get(
+		fbo.id(), ptr, branch); err == nil {
+		return block, nil
+	}
+	if block, err := fbo.config.BlockCache().Get(ptr); err == nil {
+		fbo.config.BlockOps().Prefetcher().PrefetchAfterBlockRetrieved(block, kmd, defaultOnDemandRequestPriority)
```

Comment:
Why is this needed?  If we're getting the block from the cache, shouldn't the prefetching have already happened, when the block was put in the cache in the first place?  I guess the difference is the priority?  But I'm not sure why this should be high priority, since no one has actually asked for the next level of data yet.

If you keep this, please split this line after "PrefetchAfterBlockRetrieved(" and a comment for why it's needed.